### PR TITLE
Introduce Polls

### LINKED ISSUE #4: Polls/Surveys
It must be possible to create polls or surveys on the page.

### PR DESCRIPTION
**Pull request by Bartzi**
fixes #4 

you can create new polls in the admin area.
If you want to see all polls go to: `/polls`

### CODE REVIEW

**Comment by Bartzi** [comment]
okay.. I've renamed the view `view_all` to `list`, I also fixed a Javascript problem :smile:

**Comment by invliD** [comment]
:+1:

File: _1327/polls/models.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		)
+
+
+class Choice(models.Model):
+	poll = models.ForeignKey(Poll, related_name="choices")
+	text = models.CharField(max_length=255)
+	description = models.TextField(default="", blank=True)
+	votes = models.IntegerField(default=0)
+
+	def __str__(self):
```
Comment (janno42) [inline _1327/polls/models.py]: please use `__unicode__`

Comment (invliD) [inline _1327/polls/models.py]: Don't. ``__str__`` is fine.

File: _1327/polls/templates/polls_index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+{% endblock %}
+
+{% block content %}
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <div class="panel-title">
+                {% trans "Polls you can vote for" %}
+            </div>
+        </div>
+        {% if running_polls|length ==  0 %}
```
Comment (janno42) [inline _1327/polls/templates/polls_index.html]: too much space ;)

File: _1327/polls/templates/polls_index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        {% endif %}
+    </div>
+
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <div class="panel-title">
+                {% trans "Results of Polls" %}
+            </div>
+        </div>
+        {% if finished_polls|length ==  0 %}
```
Comment (janno42) [inline _1327/polls/templates/polls_index.html]: and again

File: _1327/polls/templates/polls_index.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        <div class="panel-body">
+            {% trans "There are no results you can see." %}
```
Comment (janno42) [inline _1327/polls/templates/polls_index.html]: can you please format all such messages in italic?

File: _1327/polls/views.py [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+from _1327.user_management.shortcuts import get_object_or_error
+
+
+def list(request):
+	running_polls = []
+	finished_polls = []
+	# do not show polls that start in the future and that a user is not allowed to see
+	for poll in Poll.objects.filter(start_date__lte=datetime.date.today()):
+		if request.user.has_perm(Poll.VIEW_PERMISSION_NAME):
+			if datetime.date.today() < poll.end_date and not poll.participants.filter(id=request.user.pk).exists():
```
Comment (janno42) [inline _1327/polls/views.py]: what about polls that end today?

**Comment by janno42** [comment]
the choices must be sortable or at least in the order in which they have been created. if you want to create a new issue for that it should be required for launch.

**Comment by janno42** [comment]
the descriptions of the votes aren't shown anywhere during voting. is that correct?

**Comment by janno42** [comment]
the description of the poll is also not shown during voting, only on the results page.

**Comment by Bartzi** [comment]
the description of the votes is not shown because I'm not sure where to put that and I actually do not want to do that. I will create a new issue for that. I will also create a new issue for making choices sortable

**Comment by janno42** [comment]
:+1:

**Pull request merged**